src/cbc_sdk/platform/observations.py

```diff
@@ -173,6 +173,14 @@ def _get_detailed_results(self):
                 message="user-specified timeout exceeded while waiting for results"
             )
 
+    def get_network_threat_metadata(self):
+        """Requests Network Threat Metadata.
+
+        Returns:
+            NetworkThreatMetadata: Get the metadata for a given detector (rule).
+        """
+        return NetworkThreatMetadata(self._cb, self.rule_id)
```

Comment:
I believe this may be `tms_rule_id` instead of `rule_id`. Though need to confirm with real data

Reply:
Note not every observation type will support this look up only observations related to NTA and IDS. We may actually only want to do this on alerts instead of observations where `tms_rule_id` is definitely available

Reply:
The name is based on real data. Also, does this mean we should move it out of the observations? Should we move it to a separate python module?

Reply:
This will be useful for determining if the type of observation or alert will have threat metadata as rule_id will be populated for all types in the future which won't work for threat metadata.

This also needs to be exposed in the Alert class too so it will probably need a separate file to prevent an infinite import loop